backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -215,8 +216,11 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
         |(
         |cd $cwd
         |INSTANTIATED_COMMAND
-        |)
+        |) > $stdoutPath 2> $stderrPath
```

Comment:
need to capture to controlled filenames in the main script, not just the wrapper script that's only for shared filesystems.

Reply:
Mea culpa!